Client/mods/deathmatch/logic/CClientModel.cpp

```diff
@@ -78,30 +78,49 @@ bool CClientModel::Allocate(ushort usParentID)
     return false;
 }
 
+// You can call it only in destructor for DFF.
```

Comment:
Shouldn't you mark this method private in such case?

Reply:
It is still possible to use it with TXD